csi/controllerserver.py

```diff
@@ -49,7 +51,7 @@ def execute_gluster_quota_command(privkey, user, host, gvolname, path, size):
         "%s" % gvolname,
         "limit-usage",
         "/%s" % path,
-        "%s" % size * 0.05,
+        "%s" % str(size),
```

Comment:
nit: `str` is not required here

Reply:
- ya, just wanted to be explicit (as it anyways will be turned to string).
- will remove it